packages/expo-cli/src/commands/upload/submission-service/SubmissionService.types.ts

```diff
@@ -0,0 +1,41 @@
+import { AndroidSubmissionConfig } from './android/AndroidSubmissionConfig';
```

Comment:
do we use a naming convention like that anywhere else?
`SubmissionService.types.ts`

Reply:
Yes, we do:
<img width="683" alt="Screenshot 2020-05-26 at 11 20 09" src="https://user-images.githubusercontent.com/5256730/82883549-e103c500-9f42-11ea-9a43-c20381e45a54.png">